README.md

```diff
@@ -21,6 +21,10 @@ See how it looks like (on Sentinel chain, which is cosmos-sdk based chain, and N
 Download the latest release from [the releases page](https://github.com/QuokkaStake/tmtop/releases/). After that, you should unzip it, and you are ready to go:
 
 ```sh
+# auto
+wget $(curl -s https://api.github.com/repos/QuokkaStake/tmtop/releases/latest | grep "browser_download_url.*linux_amd64.tar.gz" | cut -d '"' -f 4) -O tmtop_linux_amd64_latest_release.tar.gz && tar zxvf tmtop_linux_amd64_latest_release.tar.gz && cp tmtop /usr/local/bin/ && tmtop
```

Comment:
Another option I found, seems more fancy, what do you think?
```
curl https://api.github.com/repos/QuokkaStake/tmtop/releases/latest | jq -r '.assets[] | select (.name | endswith("linux_amd64.tar.gz")) | .browser_download_url'
```

Reply:
Please feel free to merge the code. I did consider using jq, but I was worried that the server might not have jq installed and the download would fail, so I chose the current solution.

Regarding the parts in the TODO list, I will write a complete bash script when I have time, which will include a comprehensive detection scheme.